aana/integrations/external/decord.py

```diff
@@ -182,7 +182,7 @@ def generate_frames(
         batch_timestamps = timestamps[i : i + batch_size]
         yield FramesDict(
             frames=batch_frames,
-            frame_ids=list(range(len(batch_frames))),
+            frame_ids=list(range(i, i + len(batch_frames))),
```

Comment:
Should we fix the media_id in the line 179 too.

Reply:
Good catch! Fixed that as well.